crates/rolldown/src/stages/link_stage/tree_shaking.rs

```diff
@@ -116,8 +116,15 @@ fn include_symbol(ctx: &mut Context, symbol_ref: SymbolRef) {
   let canonical_ref_symbol = ctx.symbols.get(canonical_ref);
   let mut canonical_ref_owner = ctx.modules[canonical_ref.owner].as_ecma().unwrap();
   if let Some(namespace_alias) = &canonical_ref_symbol.namespace_alias {
+    if let Some(named_import) = canonical_ref_owner.named_imports.get(&canonical_ref) {
+      let import = &canonical_ref_owner.import_records[named_import.record_id];
+      include_statement(ctx, canonical_ref_owner, import.stmt_idx + 1);
```

Comment:
`0` index is preserved, so `+ 1` to get the real `stmt_info`

Reply:
include the statement, https://github.com/rolldown/rolldown/pull/1714/files#diff-035ce9006e573c9e7d8ea10510bcf1b73f4fa35cd515dd88df96c1d36cd34581L345-L346, we need to explicitly include now, because the side effects is not always `true` now.